GameData/RealSolarSystem/Compatibility/CustomAsteroids.cfg

```
@@ -673,4 +673,297 @@ AsteroidSets
 			max = -2
 		}
 	}
+	
+	ASTEROIDGROUP
+	{
+		name = innerOort
+		title = Inner Oort Cloud Obj.
+
+		centralBody = Sun
+
+		// One object every 710 Earth days or about 2 years
+		spawnRate = 0.00141
+
+		orbitSize
+		{
+			type = SemimajorAxis
+			min = Ratio(Earth.sma, 2500)
+			max = Ratio(Earth.sma, 20000)
+		}
+
+		eccentricity
+		{
+			dist = Uniform
+			min = 0.1
+			max = 0.999
+		}
+
+		inclination
+		{
+			dist = Uniform
+			min = -20
+			max = 20
+		}
+
+		asteroidTypes
+		{
+			key = 0.8 CaAsteroidIcy
+			// key = 0.1 CaCometActive
+			key = 0.18 PotatoComet // CaCometInterstellar
+			key = 0.02 PotatoRoid
+		}
+
+		orbitType = long
+
+		sizes 
+		{
+			key = 0.3 G
+			key = 0.5 H
+			key = 0.2 I
+		}
+	}
+	
+	ASTEROIDGROUP
+	{
+		name = outerOort
+		title = Outer Oort Cloud Obj.
+
+		centralBody = Sun
+
+		// One object every 1825 Earth days or about 5 years
+		spawnRate = 0.00055
+
+		orbitSize
+		{
+			type = SemimajorAxis
+			min = Ratio(Earth.sma, 15000)
+			max = Ratio(Earth.sma, 50000)
+		}
+
+		eccentricity
+		{
+			dist = Uniform
+			min = 0.04
+			max = 0.9999
+		}
+
+		inclination
+		{
+			dist = Uniform
+			min = -180
+			max = 180
+		}
```

I recommend an isotropic distribution, which was specifically designed for Oort objects. An inclination that's been uniformly drawn in 1-D will produce a 2-D distribution of orbital planes that is biased towards low-inclination orbits (specifically, with a 2-D density proportional to `1/|sin i|`).
```suggestion
		inclination
		{
			dist = Isotropic
		}
```